apps/web/package.json

```diff
@@ -1,37 +1,42 @@
 {
   "name": "web",
-  "version": "0.0.0",
+  "version": "0.0.1",
+  "private": true,
   "scripts": {
-    "dev": "vite dev --port 3000",
+    "dev": "vite dev",
```

Comment:
```suggestion
    "dev": "vite dev --port 3000",
```

Reply:
Ah I already set it in the vite config itself :)

Reply:
Yesss!

https://vitejs.dev/config/server-options.html